### ISSUE TITLE
Use relative links for images and other pages

### ISSUE THREAD

**Issue by agitter** [gitter-lab/ml-bio-workshop#74]
Some of our current links are hard-coded.  We can convert these to relative links to make it easier to adapt and port this workshop.

**Comment by cmilica** [comment]
Should we do this for this workshop?

**Comment by agitter** [comment]
No, this is a low priority change to make after the workshop.  It won't affect the rendered HTML at all.

**Comment by agitter** [comment]
As part of this update, we can move all of the lesson images from `assets` into `fig` so we can find them all in one place.
